ext/sockets/tests/bug46360.phpt

```diff
@@ -3,12 +3,6 @@ Bug 46360 - TCP_NODELAY constant (sock_get_option, sock_set_option)
 --CREDITS--
 Florian Anderiasch
 fa@php.net
---SKIPIF--
-<?php
-    if (version_compare(phpversion(), '5.2.7', '<')) {
-        die('skip old php, not eligible');
-    }
-?>
 --FILE--
 <?php
     var_dump('TCP_NODELAY');
```

Comment:
Not really related, but this test looks broken. It probably wants to test the `TCP_NODELAY` constant, not the string :D